zerver/context_processors.py

```diff
@@ -101,10 +101,12 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
         realm_uri = settings.ROOT_DOMAIN_URI
         realm_name = None
         realm_icon = None
+        realm_is_self_hosted = True
     else:
         realm_uri = realm.uri
         realm_name = realm.name
         realm_icon = get_realm_icon_url(realm)
+        realm_is_self_hosted = realm.plan_type == 1
```

Comment:
It would be better to use the constant defined in `models.py`, `Realm.PLAN_TYPE_SELF_HOSTED`, instead of the number because someone reading the code will understand or be able to reference what that is. If it's just the integer, who knows why that integer is the correct one to be checking here.

Reply:
oh yes i got it